pkg/collector/listeners/docker.go

```diff
@@ -329,21 +307,57 @@ func (s *DockerService) GetADIdentifiers() ([]string, error) {
 }
 
 // GetHosts returns the container's hosts
-// TODO
 func (s *DockerService) GetHosts() (map[string]string, error) {
-	return s.Hosts, nil
+	if s.Hosts != nil {
+		return s.Hosts, nil
+	}
+
+	ips := make(map[string]string)
+	c := docker.Container{ID: string(s.ID)}
+
+	cInspect, err := c.Inspect(false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to inspect container %s", string(s.ID)[:12])
+	}
+	for net, settings := range cInspect.NetworkSettings.Networks {
+		ips[net] = settings.IPAddress
+	}
+	if _, found := ips["bridge"]; !found && cInspect.NetworkSettings.IPAddress != "" {
```

Comment:
I think you can remove this chunk, `NetworkSettings.IPAddress` is deprecated

Reply:
dang you're right, and I gave you a card about this 🤦‍♂️